### PR TITLE
Adds gitignore, pagakage.json, edits units from em to ex

### PR DESCRIPTION
**Pull request by andresgalante**
Hi Sarah, this adds pagakage.json so people can run `npm install` and then `gulp` to develope the project.

Next step is to improve the README so it shows what it does and it's clear on how to use. Maybe add a CDN for the CSS file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/srambach/classbxr.git"
+  },
+  "keywords": [
+    "CSS",
+    "HTML"
+  ],
+  "author": "Sarah Rambach",
```
Comment (srambach) [inline package.json]: "Rambacher"  :-)

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Sarah Rambach
```
Comment (srambach) [inline LICENSE]: "Rambacher"  :-)

File: less/classbxr.less [hunk 3]
```diff
@@ -12,9 +12,9 @@
     background-color: white;
     color: cornflowerblue;
     position: absolute;
-    top: -.5em;
+    top: -1.5ex;
     white-space: nowrap;
-    font-family: sans-serif;
+    font-family: monospace;
```
Comment (srambach) [inline less/classbxr.less]: The thing about mono is that it's wider - I'd like to keep this as tight as possible so it minimizes the disruption to the layout.

Comment (andresgalante) [reply less/classbxr.less]: Ok, I'll change it back. I though that since it's code then `monospace` it's more appropriate.

**Comment by srambach** [comment]
Great!! Thanks, @andresgalante !!

**Pull request merged**